cmd/nonce-service/config.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think this file should be in `test/config/` no?

Reply:
Yup, was just being a bit lazy.